src/server.c

```diff
@@ -4995,6 +4999,8 @@ sds genRedisInfoString(const char *section) {
             "expired_time_cap_reached_count:%lld\r\n"
             "expire_cycle_cpu_milliseconds:%lld\r\n"
             "evicted_keys:%lld\r\n"
+            "eviction_exceeded_time:%lld\r\n"
+            "current_eviction_exceeded_time:%lld\r\n"
```

Comment:
how about?
```suggestion
            "total_eviction_exceeded_time:%lld\r\n"
            "current_eviction_exceeded_time:%lld\r\n"
```

Reply:
do we really want to expose `current_eviction_exceeded_time`? (which is actually a unix timestamp)

i think we should only expose `total_eviction_exceeded_time` and rename it to `used_mem_above_maxmemory_time` and maybe have it in the `# Memory` section in INFO

Reply:
I use `current_eviction_exceeded_time` to make the metric real time. But before `call` `performEvication` is called, so maybe it is not so necessary. For others, WDYT @oranagra.

Reply:
@guybe7 it's not a unix time, it's a delta. `now - <time when we went above the limit>`
@huangzhw i didn't understand your last comment.

Reply:
@oranagra Sorry, I made a mistake. I mean `server.stat_last_eviction_exceeded_time` is a timestamp. With this make `current_eviction_exceeded_time` and `total_current_eviction_exceeded_time` real time updated.